addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -290,6 +295,100 @@ database, without the s_newsletter_list class. See fixNewsletterListClass.
+                                <div class="s_hr pt8 pb24" data-snippet="s_hr" data-name="Separator">
+                                    <hr class="s_hr_1px s_hr_solid"/>
+                                </div>
+                                <t groups="website.group_website_designer">
+                                    <t t-set="mailing_lists" t-value="request.env['mailing.list'].search([('is_public', '=', True)])"/>
+                                    <div t-attf-class="#{'' if len(mailing_lists) > 1 else 'd-none'} col-lg-12 s_website_form_field s_website_form_model_required pb24" t-if="len(mailing_lists) > 0" data-type="many2many" data-name="Field">
+                                        <label class="col-sm-auto s_website_form_label d-none" style="width: 250px" for="mailing_list_">
+                                            <span class="s_website_form_label_content">Our mailing lists</span>
+                                            <span class="s_website_form_mark"> *</span>
+                                        </label>
+                                        <div class="s_website_form_multiple" data-name="list_ids" data-display="vertical">
+                                            <t t-foreach="mailing_lists" t-as="record">
+                                                <div class="checkbox col-12">
+                                                    <div class="form-check">
+                                                        <input type="checkbox" class="s_website_form_input form-check-input" name="list_ids"
+                                                            t-attf-id="mailing_list_#{record.id}" t-att-checked="len(mailing_lists) &lt;= 1" t-att-value="record.id" required="1"/>
+                                                        <label class="form-check-label s_website_form_check_label" t-attf-for="mailing_list_#{record.id}">
+                                                            <t t-esc="record.name"/>
+                                                        </label>
+                                                    </div>
+                                                </div>
+                                            </t>
+                                        </div>
+                                        <div class="s_website_form_field_description small form-text text-muted">
+                                            We send one weekly newsletter per list and always try to keep it interesting. You can unsubscribe at any time.
+                                        </div>
+                                    </div>
+                                </t>
+                                <div class="s_website_form_submit s_website_form_no_submit_label col-12" data-name="Submit Button">
+                                    <div style="width: 250px;" class="s_website_form_label"/>
+                                    <span id="s_website_form_result"></span>
```

Comment:
```suggestion
                                    <span id="s_website_form_result"/>
```

Also, I think it should be placed after the button, so the error message won't push the submit button :
<img width="784" height="237" alt="Capture d’écran 2025-10-02 à 11 22 53" src="https://github.com/user-attachments/assets/2f59f3e5-0755-4b63-8698-47bf4a61cd52" />